.elasticbeanstalk/config.yml

```diff
@@ -0,0 +1,20 @@
+branch-defaults:
```

Comment:
This file shouldn't be uploaded since it's a personal configuration for elastic beanstalk. Add .elasticbeanstalk to .gitignore please

Reply:
I deleted .elasticbeanstalk/config.yml